app/controllers/submissions_controller.rb

```diff
@@ -491,6 +491,46 @@ def resubmit_to_turnitin
     end
   end
 
+   def vericite_report
```

Comment:
We don't mind the separate route, but since this function and the turnitin version are 99% the same, a combined method seems to be more generically going forward.